reporters/index.js

```diff
@@ -138,11 +146,29 @@ const Poll = async function(callback) {
     // UNISWAP_PRICE_FEED_CONFIG={"type":"uniswap","twapLength":86400,"lookback":7200,"invertPrice":true,"uniswapAddress":"0x1e4F65138Bbdb66b9C4140b2b18255A896272338"}
     const uniswapPriceFeedConfig = JSON.parse(process.env.UNISWAP_PRICE_FEED_CONFIG);
 
-    // Change `periodLengthSeconds` to modify how far back the report will look for its shorter report period. For example, setting this to
-    // `24 * 60 * 60` means that the report will include aggregated data for the past 24 hours.
-    const periodLengthSeconds = process.env.PERIOD_REPORT_LENGTH ? process.env.PERIOD_REPORT_LENGTH : 24 * 60 * 60;
-
-    await run(empAddress, walletsToMonitor, referencePriceFeedConfig, uniswapPriceFeedConfig, periodLengthSeconds);
+    // The report will always display "cumulative" and "current" data but it will also show data for a shorter period ("period") whose
+    // start and end dates we can control:
+
+    // Change `endDateOffsetSeconds` to modify the end date for the "period". End date will be (now - endDateOffsetSeconds).
+    const endDateOffsetSeconds =
+      process.env.PERIOD_END_DATE_OFFSET && process.env.PERIOD_END_DATE_OFFSET > 0
+        ? Math.round(process.env.PERIOD_END_DATE_OFFSET)
+        : 0;
+
+    // Change `periodLengthSeconds` to modify the "period" start date. Start date will be (endDate - periodLengthSeconds).
+    const periodLengthSeconds =
+      process.env.PERIOD_REPORT_LENGTH && process.env.PERIOD_REPORT_LENGTH > 0
+        ? Math.round(process.env.PERIOD_REPORT_LENGTH)
+        : 24 * 60 * 60;
```

Comment:
This is probably a question for @hlambur, but I just want to double check that we're in agreement that this is the most intuitive way to provide these inputs.

Did we consider allowing the user to provide a start datetime and end datetime string? I'm not super opinionated here, so if others are cool with this, I'm happy to ship it :).

Reply:
if we allow a start/end date time, then we need to write code to parse that. I think this feature will only be used by "pro" users and the simpler the better.